fda/datasets/_real_datasets.py

```diff
@@ -0,0 +1,234 @@
+
+import textwrap
+
+import numpy as np
+import rdata
```

Comment:
I get the following error from the rdata package when I try to import  ***fda.datasets***:

>$ python
Python 3.6.5 |Anaconda, Inc.| (default, Apr 26 2018, 08:42:37) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
***>>> import fda.datasets***
Traceback (most recent call last):
>>File "<stdin>", line 1, in <module>
  File "/Users/pablomm/Functional data analysis/fda-GAA-UAM/fda/__init__.py", line 23, in <module>
    from . import datasets, covariances
  File "/Users/pablomm/Functional data analysis/fda-GAA-UAM/fda/datasets/__init__.py", line 1, in <module>
    from ._real_datasets import (fdata_constructor, fetch_cran_dataset,
  File "/Users/pablomm/Functional data analysis/fda-GAA-UAM/fda/datasets/_real_datasets.py", line 5, in <module>
    import rdata
  File "/Users/pablomm/anaconda3/lib/python3.6/site-packages/rdata-0.2.1-py3.6.egg/rdata/__init__.py", line 1, in <module>
    from . import conversion
  File "/Users/pablomm/anaconda3/lib/python3.6/site-packages/rdata-0.2.1-py3.6.egg/rdata/conversion/__init__.py", line 1, in <module>
    from ._conversion import (RExpression, RLanguage,
  File "/Users/pablomm/anaconda3/lib/python3.6/site-packages/rdata-0.2.1-py3.6.egg/rdata/conversion/_conversion.py", line 10, in <module>
    import xarray
  File "/Users/pablomm/anaconda3/lib/python3.6/site-packages/xarray-0.10.9-py3.6.egg/xarray/__init__.py", line 10, in <module>
    from .core.alignment import align, broadcast, broadcast_arrays
  File "/Users/pablomm/anaconda3/lib/python3.6/site-packages/xarray-0.10.9-py3.6.egg/xarray/core/alignment.py", line 11, in <module>
    from .indexing import get_indexer_nd
  File "/Users/pablomm/anaconda3/lib/python3.6/site-packages/xarray-0.10.9-py3.6.egg/xarray/core/indexing.py", line 11, in <module>
    from . import duck_array_ops, nputils, utils
  File "/Users/pablomm/anaconda3/lib/python3.6/site-packages/xarray-0.10.9-py3.6.egg/xarray/core/duck_array_ops.py", line 22, in <module>
    from . import dask_array_compat
  File "/Users/pablomm/anaconda3/lib/python3.6/site-packages/xarray-0.10.9-py3.6.egg/xarray/core/dask_array_compat.py", line 47, in <module>
    AxisError = np.AxisError
AttributeError: module 'numpy' has no attribute 'AxisError'

It is needed a specific version of numpy or xarray to import rdata not included in the setup.py?

Reply:
My version of numpy
>*>>> import numpy as np*
*>>> np._ _ version _ _*
***'1.12.1'***

Reply:
Solved after update numpy. It might be a good idea to add to the setup requirements the necessary minimum version of the packages.